app/screens/settings/notification_settings_email/notification_settings_email_base.js

```diff
@@ -84,25 +84,25 @@ export default class NotificationSettingsEmailBase extends PureComponent {
         }
     };
 
-    setEmailNotifications = (value) => {
-        const {sendEmailNotifications} = this.props;
-
-        let email = 'false';
-        if (sendEmailNotifications && value !== Preferences.INTERVAL_NEVER.toString()) {
-            email = 'true';
-        }
-
-        this.setState({
-            email,
-            newInterval: value,
-        });
+    setEmailInterval = (value) => {
+        this.setState({newInterval: value});
     };
 
     saveEmailNotifyProps = () => {
-        const {actions, currentUser} = this.props;
-        const {email, emailInterval, newInterval} = this.state;
+        const {emailInterval, newInterval} = this.state;
 
         if (emailInterval !== newInterval) {
+            const {
+                actions,
+                currentUser,
+                sendEmailNotifications,
+            } = this.props;
+
+            let email = 'false';
+            if (sendEmailNotifications && newInterval !== Preferences.INTERVAL_NEVER.toString()) {
+                email = 'true';
+            }
```

Comment:
This actually fix the issue when email state is still not set for the first time.  Hitting save will result to `state.email` being null which ends up as no email notify props saved on the server.  
Instead of setting `email` on state (which does not makes sense since it's not for rendering the component), determine its value here during saving.